static/app/components/metrics/metricSearchBar.tsx

```diff
@@ -129,7 +129,6 @@ export function MetricSearchBar({
       placeholder={t('Filter by tags')}
       query={query}
       savedSearchType={SavedSearchType.METRIC}
-      disallowWildcard
```

Comment:
Just to double check: This component is also used in alerts, where we use the event-stats endpoint.
Will is work also there?